restore.c

```diff
@@ -799,6 +806,33 @@ create_recovery_signal(void)
 	}
 }
 
+static void
+remove_standby_signal(void)
+{
+	struct stat stat_buf;
+	char path[MAXPGPATH];
+
+	if (verbose && !check)
+		printf(_("----------------------------------------\n"));
+
+	elog(INFO, _("removing standby.signal if exists"));
+
+	if (!check)
+	{
+		snprintf(path, lengthof(path), "%s/standby.signal", pgdata);
+
+		if (stat(path, &stat_buf) == 0)
+		{
+			if (remove(path))
+			{
+				elog(ERROR, _("could not remove \"%s\": %s"), path,
+					strerror(errno));
+			}
+			elog(INFO, _("removed standby.signal"));
```

Comment:
How about adding a little more warning message(info level)?
For example, "SR configuration has been released and operates in standalone mode. If SR configuration is required, a new setting is required." I think it would be better to add a message like this.
It would be a think to inform users who did not expect this behavior "Why that released the SR configuration."

Reply:
Yes, I agree. I'll change it